Spanner/src/Session/CacheSessionPool.php

```diff
@@ -660,7 +673,8 @@ private function createSessions($count)
                 $res = $this->database->connection()->batchCreateSessions([
                     'database' => $this->database->name(),
                     'sessionTemplate' => [
-                        'labels' => isset($this->config['labels']) ? $this->config['labels'] : []
+                        'labels' => isset($this->config['labels']) ? $this->config['labels'] : [],
+                        'creator_role' => isset($this->config['databaseRole']) ? $this->config['databaseRole'] : null
```

Comment:
Can we rename creator_role to database_role everywhere?

Reply:
And the name database role specifically is shared from all the other languages
eg Ruby implementation here: https://github.com/googleapis/google-cloud-ruby/pull/19067/files#diff-78756a65b23b98e0db00dd49fb26d337de20dda6c9490c38e9ffef646f4ac96cR299